.orchestra/support/mass-testing/generate_inclusions

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import argparse
+import re
+import sqlite3
+from pathlib import Path
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input", help="Input database (main.db)")
+    parser.add_argument("output", help="Output yml file")
+    args = parser.parse_args()
+
+    includes = set()
+
+    if Path(args.output).exists():
+        with open(args.output) as f:
+            data = yaml.safe_load(f)
+        includes.update(data.get("include", []))
```

Comment:
What else do we have in this file?
I wonder if this script could have been an invocation to `sqlite3` emitting a CSV.

Reply:
Can we avoid this script with a one-liner with `sqlite3` dumping JSON + `jq`?

Reply:
Eh, not really, there's also `re.escape` in the mix which doesn't have a cli equivalent AFAIK